frontend/src/actions/SearchRides.js

```diff
@@ -0,0 +1,24 @@
+import {getRidesSuccess, getRidesError, getRidesPending} from "../reducers/SearchRidesReducer";
+
+const xurl = 'http://localhost:5000/api/rides?query=%7B%22originCoords%22%3A%5B-119.159392%2C34.164958%5D%2C%22beginDate%22%3A%222020-07-23T20%3A00%3A00.000Z%22%2C%22endDate%22%3A%222020-07-23T21%3A00%3A00.000Z%22%2C%22distance%22%3A5%7D'
```

Comment:
Hard-coded? I think you should stringify the data and append it to the back of the query. Since every query is different